src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -817,7 +817,7 @@ namespace OpenRCT2::Ui::Windows
                         ViewportInteractionItem::LargeScenery);
 
                     auto info = GetMapCoordinatesFromPos(screenCoords, interactionFlags);
-                    if (info.SpriteType != ViewportInteractionItem::None)
+                    if (info.EntityType != ViewportInteractionItem::None)
```

Comment:
Renaming it `EntityType` very heavily suggests that it takes a member of the `EntityType` enum class, which is not the case. Not to mention that `ViewportInteractionItem` has many member that don’t relate to entities at all, like `Terrain` and `Ride.`. I suggest renaming it to `type` or something like that. (Lower case, as we adopted earlier this year.)

Reply:
Perhaps it should just be `interactionType`, per thread actual type involved.

I just happened across it in this PR due to the name conflict with the Peep property, but perhaps this is best tackled in a separate PR, anyway.

Reply:
It’s okay to tackle in here, it isn’t hard for me to review.

If you prefer `interactionType`, that’s okay with me too.

Reply:
Alright, I've renamed the property to `interactionType`.